src/Controller/ViewController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\View;
+use App\Repository\ViewRepository;
+use App\Service\ViewHelperService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Exception\SessionNotFoundException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/admin/view')]
+class ViewController extends AbstractController
+{
+    private const SESSION_KEY = self::class;
+    private const CREATEFORM_LAST_STEP = 3;
+
+    #[Route(['', '/list'], name: 'app_view_list')]
+    public function list(Request $request, ViewRepository $viewRepository): Response
+    {
+        return $this->render('view/list.html.twig', [
+            'views' => $viewRepository->findAll(),
+        ]);
+    }
+
+    #[Route('/add', name: 'app_view_add')]
+    public function add(Request $request, ViewRepository $viewRepository, ViewHelperService $viewHelperService): Response
+    {
+        // Create session for multistep form.
+        try {
+            $session = $request->getSession();
+        } catch (SessionNotFoundException $e) {
+            $session = new Session();
+            $session->start();
+        }
+
+        $steps = $viewHelperService->getCreateFormMultiSteps();
+        $data = $session->get(self::SESSION_KEY);
+
+        // Initialize multistep form.
+        if (empty($data)) {
+            $data = $viewHelperService->getCreateFromInitValues();
+            $session->set(self::SESSION_KEY, $data);
+        }
+
+        $form = $this->createForm($steps[$data['current_step']]['class'], $data['view']);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            // If submitting on last step we save the data.
+            if (self::CREATEFORM_LAST_STEP === $data['current_step']) {
+                if ($form->isValid()) {
+                    $session->invalidate();
+
+                    $data['view']->setCreated(new \DateTimeImmutable());
```

Comment:
This should be set automatically by the entity extending https://github.com/itk-dev/economics/blob/develop/src/Entity/AbstractBaseEntity.php